packages/react-scripts/scripts/eject.js

```diff
@@ -114,13 +115,28 @@ inquirer
       fs.mkdirSync(path.join(appPath, folder));
     });
 
+    let addtlDeps = new Map();
```

Comment:
No harm in longer names; I'll change it. 😄

If I could have `'` it'd be nice: add'l., addt'l.

addl seemed too (??), so I opted for addtl.